tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": false,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
'TypeScript will type check the code you specifically refer to in your app’s source code.' are we sure we want this?

Reply:
resolved in dm